doc/migrations.md

```diff
@@ -0,0 +1,30 @@
+# Database migrations
+
+## Setting up
+
+If you are planning to use database migrations, you need to let
+Alembic know the current state of the database.
+
+Presuming you've just started off by running `make load_demo_data`,
+tell Alembic that you are on the latest database schema:
+
+```
+PYTHONPATH=. alembic stamp head
```

Comment:
Why is `PYTHONPATH=.` needed if this command is being run from `skyportal/`?

Reply:
Not sure; doesn't work without it, though.